src/features/profile/Profile.tsx

```diff
@@ -12,9 +12,9 @@ import {Routes} from '../../navigation/Router';
 import {logoutHandler} from '../../reducers/authReducer';
 import {AppState} from '../../reducers/RootReducer';
 import {testData} from '../../services/authService';
-import {setTheme} from '../../actions/themeActions';
 import {Themes} from '../../theme';
 import {RadioButton} from '../../components/radioButton/radioButton';
+import {themePicker} from '../../reducers/themeReducer';
```

Comment:
not sure is themePicker is action, i mean picker sounds like component name ...
For actions: fetchFeeds, setTheme ... etc
In my opinion themePicker is not valid name for action :(